apps/src/templates/sectionAssessments/FreeResponsesSurveyTable.story.jsx

```diff
@@ -13,8 +13,3 @@ export const SurveyOne = Template.bind({});
 SurveyOne.args = {
   freeResponses: surveyOne,
 };
-
-export const SurveyTwo = Template.bind({});
-SurveyTwo.args = {
-  freeResponses: surveyTwo,
-};
```

Comment:
This seems fair - I think it is just trying to show that if 3 different people give the exact same answer ("Code.org is the best") that each response will get its own line (instead of one line being shown)

Reply:
To clarify, it seems fair to remove this.

Reply:
Ahh makes sense